NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/service/LoggerIntercepter.kt

```diff
@@ -0,0 +1,21 @@
+package com.neuroid.tracker.service
```

Comment:
It seems like we have this across all out flavors, can we not put this in the main lib?

Reply:
Unfortunately no. The ReactNative and AndroidLib Retrofit RequestBody implementations are slightly different. The ReactNative lib has deprecated RequestBody.create() and added an extension to String called String.toRequestBody() that creates the requestBody from the event String. Same story for LoggerInterceptor. We need to reference items in the RequestBody slightly differently in Native vs ReactNative ( ReactNative RequestBody.url  vs Native RequestBody.url() ).

Reply:
was able to move to main, removed from flavors. explicitly added okhttp client version so both native and ReactNative use the same OKHTTP version.